diff/OSBB/contracts/osbb.sol

```diff
@@ -0,0 +1,63 @@
+pragma solidity >=0.4.21 <0.6.0;
+import "../node_modules/@openzeppelin/contracts/ownership/Ownable.sol";
+
+contract OSBB is Ownable {
+    
+    // Resident of OSBB
+    struct User {
+        bytes32 Name;
+        uint Flat;
```

Comment:
Please, use uint8 as minimum value. max uint8 can be 255, so it is good choice (there can not be more than 255 flats). Same for floor